### ISSUE TITLE
De-compiled App

### ISSUE THREAD

**Issue by Zipsnet** [MattTW/BlinkMonitorProtocol#1]
MAtt,

     I have de-compiled the app  to pull the api's from it, https://github.com/Zipsnet/Blink, I could probably start working on it  maybe by next week. I leave you this here, just in case.

Thnx

**Comment by MattTW** [comment]
Awesome, thanks!

**Comment by dib0** [comment]
There is an unpublished api description here:
http://prod.immedia-semi.com/api

Just for your information.

I just can't get the curl commands to work... in the API docs there is a mention of a DEV_KEY. Do the curl calls still work with you guys?

**Comment by mifbody** [comment]
HTTP calls work for me still -- I have Blink flows in Node Red application thanks to @MattTW 's documentation and I'm still getting responses on my calls.


![image](https://cloud.githubusercontent.com/assets/12959936/15858939/e9e83030-2c90-11e6-8939-d3ad392a33e3.png)

**Comment by dib0** [comment]
Thanks for your reaction. The strange thing is that logging in works for me like a charm. I do get a token, but any subsequent calls fail and I get a "Not autorised".

I just added my initial implementation to github: https://github.com/dib0/BlinkPHP
If anyone can take a look and help me further, it would be greatly appreciated. Also, the commandline/curl variant doesn't work for me either.

**Comment by dstrickler** [comment]
@dib0: I'm working on a PHP version of this as well, and having trouble too. If I get anything working, I'll try and get it uploaded so you can take a look at it.

If you've made any progress - please share!

**Comment by dib0** [comment]
Thanks!

I have a working version now. The problem I had was that I logged in on a US region and worked on a non-US URL after that.
By logging into a different URL I stayed authorised. I'm working with the python componentn in Home Assistant now.

For reference:
My PHP script: https://github.com/dib0/BlinkPHP

The problem with non-US as it was solved in the blinkpy component:
https://github.com/fronzbot/blinkpy/issues/1
